.github/workflows/nightly-verilator.yml

```diff
@@ -1,5 +1,10 @@
 name: Verilator Nightly
 on:
+  workflow_call:
```

Comment:
Check if workflow_call ref is auto-picked up

Reply:
actions/checkout@v3 may not need us to specify an explicit git ref

Reply:
I couldn`t find from the documentation a clear answer to this...in the meantime, I`d propose to keep this logic as it is more explicit...

Reply:
From https://github.com/actions/checkout:

> ref: The branch, tag or SHA to checkout. When checking out the repository that triggered a workflow, this defaults to the reference or SHA for that event. Otherwise, uses the default branch.

Reply:
Also, from https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow:

> {ref} can be a SHA, a release tag, or a branch name. Using the commit SHA is the safest for stability and security. F